Samples/Shared/CameraKsPropertyHelper/FieldOfView2ConfigCapsPropertyPayload.h

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+#pragma once
+#include "FieldOfView2ConfigCapsPropertyPayload.g.h"
+#include "KSHelper.h"
+
+namespace winrt::CameraKsPropertyHelper::implementation
+{
+    struct FieldOfView2ConfigCapsPropertyPayload :
+        FieldOfView2ConfigCapsPropertyPayloadT<FieldOfView2ConfigCapsPropertyPayload>,
+        PropertyValuePayloadHolder<KsFieldOfView2ConfigCapsPropPayload>
+    {
+        FieldOfView2ConfigCapsPropertyPayload(Windows::Foundation::IPropertyValue property) :
+            PropertyValuePayloadHolder(property)
+        {
+            for (int i = 0; i < m_payload->configCaps.DiscreteFoVStopsCount; i++)
+            {
+                m_discreteFoVStops.Append(m_payload->configCaps.DiscreteFoVStops[i]);
+            }
+        }
+
+        int32_t DefaultDiagonalFieldOfViewInDegrees()
+        {
+            return m_payload->configCaps.DefaultDiagonalFieldOfViewInDegrees;
+        }
+        int32_t DiscreteFoVStopsCount()
```

Comment:
>DiscreteFoVStopsCount [](http://example.com/codeflow?start=16&length=21)

nit: I don't think we need to project this as IVectorView knows it's size and only contains the valid values from the full payload array.

Reply:
yeah sure you are right, but I prefer to project the count value returned by driver than the value we infer from the set of values returned in the array, this may help identify discrepancies from the app side.

Reply:
yeah fine to keep.